clusterutil/cluster_validation_config.go

```diff
@@ -8,20 +8,41 @@ import (
 )
 
 type ClusterValidationConfig struct {
-	Label           string                  `yaml:"label" category:"experimental"`
-	registeredFlags flagext.RegisteredFlags `yaml:"-"`
+	Label            string                  `yaml:"label" category:"experimental"`
+	AdditionalLabels flagext.StringSliceCSV  `yaml:"additional_labels" category:"experimental"`
+	registeredFlags  flagext.RegisteredFlags `yaml:"-"`
 }
 
 func (cfg *ClusterValidationConfig) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	cfg.registeredFlags = flagext.TrackRegisteredFlags(prefix, f, func(prefix string, f *flag.FlagSet) {
-		f.StringVar(&cfg.Label, prefix+"label", "", "Optionally define the cluster validation label.")
+		f.StringVar(&cfg.Label, prefix+"label", "", "Primary cluster validation label.")
+		f.Var(&cfg.AdditionalLabels, prefix+"additional-labels", "Comma-separated list of additional cluster validation labels for server-side validation.")
```

Comment:
I'd add a short explanation of what "additional labels" implies here: It's not the label that the cluster sends, but they're labels that it accepts from requests